app/controllers/pieces_controller.rb

```diff
@@ -0,0 +1,12 @@
+class PiecesController < ApplicationController
+	def show
+    @piece = Piece.find(params[:id])
+    @pieces = @piece.game.pieces # display all the pieces on the board
```

Comment:
You'll want to make sure what you show only "active" pieces so you should create a scope on the pieces model and use it in this controller

def self.active
  where(active: true)
end